config/zones/CA-ON.yaml

```diff
@@ -71,6 +73,10 @@ emissionFactors:
       - datetime: '2021-01-01'
         source: Electricity Maps, 2021 average
         value: 58.066760035663556
+    nuclear:
+      datetime: '2013-01-01'
+      source: Life cycle greenhouse gas emissions of electricity generation in the province of Ontario Canada 2013
```

Comment:
Nitpick, but could you please make it comply more with MLA style citation -> Author "Title" (Year)

I'll approve nonetheless, Good job for being the first to add new regional emission factors since the release 💪

Reply:
@pierresegonne maybe we should add what citation style we use somewhere?
I personally prefer IEEE but MLA, APA, etc are more descriptive the way we use it now.

Reply:
Sure, feel free to add to the wiki :)

Reply:
Will add it to the to-do 😅

I assume we want to go with MLA?

Reply:
@pierresegonne completed the MLA reformatting